src/zscript/damnums/damnum.txt

```diff
@@ -59,7 +60,9 @@ class DamNum : Actor
 	/*
 	 * Spawn damage numbers. Does what it says on the tin.
 	 */
-	static int spawnDamageNumbers(Actor source = null, Actor target = null, int damage = 0, Name damagetype = 'Default') {
+	static int spawnDamageNumbers(Actor source = null, Actor target = null, int damage = 0, Name damagetype = 'Default', 
+		bool usePos = false, Vector3 dnpos = (0,0,0)) 
+	{
```

Comment:
Bear in mind this will still show up and it'll look incomplete. Next time might do best as just a comment. :P